README_CN.md

```diff
@@ -49,7 +49,7 @@
 
 Stay In Tarkov (SIT) 是逃离塔科夫的一个模组, 设计目的为实现联机合作.
 
-SIT 项目目前由一个小型的开发团队进行共同开发. 虽然本项目当前处于可用状态, 但在游玩过程中, 仍然会出现一些 BUG 或是性能问题. 逃离塔科夫与 SPT-Aki 的更新较为频繁, 您的离线版帐号也会随着更新的到来, 必须进行重置.
+SIT 项目目前由一个小型的开发团队进行共同开发. 虽然本项目当前处于可用状态, 但在游玩过程中, 仍然会出现一些 BUG 或是性能问题. 《逃离塔科夫》和《SPT-AKI》更新频繁，很多时候，您的进度必须重新开始。
```

Comment:
英文版太过于简洁，容易导致理解错误，我改成方便理解的样子你又给改回去了...
本来此说明就是明确表示“每次aki更新的时候，你都要重开存档”，而玩过离线因更新删过档的基本看一眼就能理解是啥意思了，而aki更新又不需要删档的只有像3.7.0到3.7.6那样的BUG修复

Reply:
sounds like clarification is needed in the upstream EN section? upgrade doesn't always result in need of reset of profile progression. the difference here is

"more often than not,  profile restart is needed" vs "profile reset is required."

Reply:
This part of description are just clearifying the profile reset issue, because it's not a SIT problem, but AKI.
Update with only bug fixes won't require you to do a reset like recent 3.8.1. however, SIT are always trying to follow and update to the latest version from Live, but with major updates we have to use BE build from AKI, which "profile reset" is needed.

Reply:
that's fine. ready for review